common/src/main/kotlin/no/nav/su/se/bakover/common/periode/Periode.kt

```diff
@@ -26,6 +26,18 @@ data class Periode private constructor(
             }
     }
 
+    fun erPeriodenIMånederEtter(): Boolean {
+        val dagensDato = LocalDate.now()
+
+        return this.fraOgMed.isAfter(
+            LocalDate.of(
```

Comment:
Tror man skal kunne skrive `return this.fraOgMed.isAfter(dagensDato.withDayOfMonth(dagensDato.lengthOfMonth()))`
https://docs.oracle.com/javase/8/docs/api/java/time/LocalDate.html

Men kanskje vi skal lage en extension-function?

Reply:
📅